index.js

```diff
@@ -64,10 +66,8 @@ module.exports = {
     ].filter(Boolean));
   },
 
-  treeForPublic() {
-    let parentAddon = this.parent.findAddonByName(this.parent.name());
-    if (!parentAddon) { return; }
-
+  treeForPublic() { 
+    let parentAddon = this.parent.treePaths ? this.parent : this;
```

Comment:
I actually just landed a fix to give a more descriptive error for that case 🙂 
https://github.com/ember-learn/ember-cli-addon-docs/commit/bc187faae8c333ae4dda94eba5058a480863c154